### ISSUE TITLE
HSV to HSL can cause S to become NaN

### ISSUE THREAD

**Issue by phlebotinum** [dylan/Colors#19]
to reproduce:
```swift
 let colorHSV = Color(hue: 0.0, saturation: 0.0, value: 100.0)
 let colorConvertedHSL = colorHSV.converted(to: .hsl)
 print("saturation:", colorConvertedHSL.hsl.saturation)
// saturation: nan
```

**Comment by phlebotinum** [comment]
Fixed via PR #21

**Comment by dylan** [comment]
:tada: Thanks for introducing testing to the project!
